Include/internal/pycore_long.h

```diff
@@ -157,13 +157,14 @@ PyAPI_FUNC(int) _PyLong_Size_t_Converter(PyObject *, void *);
 
 /* Long value tag bits:
  * 0-1: Sign bits value = (1-sign), ie. negative=2, positive=0, zero=1.
- * 2: Reserved for immortality bit
+ * 2: Reserved for immortality bit. Set to 1 for the small ints
```

Comment:
It's not reserved any more. It's used.
```suggestion
 * 2: Set to 1 for the small ints
```